glui.cpp

```diff
@@ -292,7 +356,7 @@ void glui_keyboard_func(unsigned char key, int x, int y)
       glutSetWindow( GLUI_Master.active_control_glui->get_glut_window_id() );
 
       GLUI_Master.active_control_glui->keyboard(key,x,y);
-	  finish_drawing();
+	  _sync_drawing_by_calling_glFinish();
```

Comment:
Suggestion: Split this change off into a seperate branch/commit to enable a fast-track review and merge.